yoti-sdk-spring-boot-example/src/main/java/com/yoti/api/examples/springboot/IdentitySessionController.java

```diff
@@ -0,0 +1,124 @@
+package com.yoti.api.examples.springboot;
+
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.yoti.api.client.DigitalIdentityClient;
+import com.yoti.api.client.identity.ShareSession;
+import com.yoti.api.client.identity.ShareSessionRequest;
+import com.yoti.api.client.identity.extension.Extension;
+import com.yoti.api.client.identity.extension.LocationConstraintContent;
+import com.yoti.api.client.identity.extension.LocationConstraintExtensionBuilder;
+import com.yoti.api.client.identity.policy.Policy;
+import com.yoti.api.client.identity.policy.WantedAttribute;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/v2")
+public class IdentitySessionController {
+
+    private static final URI REDIRECT_URI = URI.create("https://localhost:8443/v2/receipt");
+
+    private final DigitalIdentityClient client;
+
+    @Autowired
+    public IdentitySessionController(DigitalIdentityClient client) {
+        this.client = client;
+    }
+
+    @GetMapping("/digital-identity-session")
+    public String identityShareSession() {
+        ShareSession session = client.createShareSession(
+                // forMinimalShare()
+                forDynamicScenarioShare()
+                // forIdentityProfileShare()
+                // forLocationExtensionShare()
```

Comment:
Are these commented out on purpose? if so then maybe leave a comment explaining why they are commented out? or how/when should the commented out parts be un-commented?

Reply:
yes, I created a quick to switch scenarios for Darren. If he wants to create more tests he is free to do so